### ISSUE TITLE
AlertsService is not defined

### ISSUE THREAD

**Issue by svoinea** [informatics-isi-edu/chaise#634]
If the "Save metadata" operation fails, then in the console log I can see the message:
```
PUT XHR https://cirm-dev.isrd.isi.edu/ermrest/catalog/1/entity/Microscopy:v_scan [HTTP/1.1 409 Conflict 46ms]

Error: AlertsService is not defined
error@https://cirm-dev.isrd.isi.edu/chaise/viewer/image-metadata/metadata.controller.js?v=1d46d9d04a0a8059fa7e0e040e642499:32:17
processQueue@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:15961:28
scheduleProcessQueue/<@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:15977:27
$RootScopeProvider/this.$get</Scope.prototype.$eval@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:17229:16
$RootScopeProvider/this.$get</Scope.prototype.$digest@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:17045:15
$RootScopeProvider/this.$get</Scope.prototype.$apply@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:17337:13
done@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:11572:36
completeRequest@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:11778:7
requestLoaded@https://cirm-dev.isrd.isi.edu/chaise/scripts/vendor/angular.js:11711:1
```

**Comment by hongsudt** [comment]
@svoinea please provide more information. Which app is this? Who should be working on this?

**Comment by svoinea** [comment]
Already talked to Jessie. It's a minor correction to be done.

**Comment by hongsudt** [comment]
@howdyjessie please fix this or tell @svoinea what need to be done.

**Comment by howdyjessie** [comment]
This has been fixed via d9ed3169054ea8d56b19d7e0a890ba880da5e590.
